src/index.js

```diff
@@ -329,6 +349,46 @@ const initialize = async () => {
       console.log(contract);
     };
 
+    /**
+     * ERC721 Token
+     */
+
+    deployCollectiblesButton.onclick = async () => {
```

Comment:
It's better practice to use `addEventListener` rather than set `onclick` directly, in case we want to add other listeners to this in the future. So I would recommend:

``` suggestion
deployCollectiblesButton.addEventListener('click', () => {
```

Reply:
will deal with that in a separate PR